uiowa/uiowa
# Block Grid Layout Overflow

Resolves #5946 

### How to Test:
1. `blt ds` csd.uiowa.edu and default.
2. `yarn workspace uids_base build`
3. Check https://csd.uiowa.ddev.site/research at smaller breakpoints to make sure the grid (cards) is equal-width.
4. On default, create a four-column section with cards and see that the grid stays equal width.
5. If the text is overflowing, don't worry about it. That will be worked on in a [follow-up issue](https://github.com/uiowa/uiowa/issues/4678).
